samples/contrib/versioned-pipeline-ci-samples/kaggle-ci-sample/visualize_html/visualize.py

```diff
@@ -0,0 +1,52 @@
+# visualizer with html
+
+def datahtml(
+    bucket_name,
+    train_file_path
+):
+    import json
+    import seaborn as sns
+    import matplotlib.pyplot as plt
+    import os
+    image_path = os.path.join('gs://', bucket_name, 'visualization.png')
+    image_url = os.path.join('https://storage.googleapis.com', bucket_name, 'visualization.png')
+    html_path = os.path.join('gs://', bucket_name, 'kaggle.html')
+    # ouptut visualization to a file
+
+    import pandas as pd
+    df_train = pd.read_csv(train_file_path)
+    sns.set()
+    cols = ['SalePrice', 'OverallQual', 'GrLivArea', 'GarageCars', 'TotalBsmtSF', 'FullBath', 'YearBuilt']
+    sns.pairplot(df_train[cols], size = 2.5)
```

Comment:
Also, it seems that the latest interface uses 'height' instead of 'size'.

Reply:
Yes, it is. Will modify it.